### PR TITLE
Firewall rules

### PR DESCRIPTION
**Pull request by majst01**

### CODE REVIEW

**Comment by vknabel** [comment]
Do we want to support reading the firewall rules from stdin? (not sure about the flag)
It would be quite handy to manually create a firewall without creating a temporary file.

```bash
$ metalctl firewall create ..mandatory args.. --firewall-rules-file - << RULES
---
egress:
- comment: allow outgoing https
  ports:
  - 443
  protocol: TCP
  to:
  - 0.0.0.0/0
RULES
```

**Comment by majst01** [comment]
> Do we want to support reading the firewall rules from stdin? (not sure about the flag) It would be quite handy to manually create a firewall without creating a temporary file.
> 
> ```shell
> $ metalctl firewall create ..mandatory args.. --firewall-rules-file - << RULES
> ---
> egress:
> - comment: allow outgoing https
>   ports:
>   - 443
>   protocol: TCP
>   to:
>   - 0.0.0.0/0
> RULES
> ```

Would be nice, but i would really like to stop spending time on this rarely used feature

File: cmd/firewall.go [hunk 1]
```diff
@@ -186,8 +241,28 @@ func (c *firewallCmd) createRequestFromCLI() (*models.V1FirewallCreateRequest, e
...
+		return nil, nil
+	}
+
+	firewallRules, err := os.ReadFile(firewallRulesFile)
+	if err != nil {
+		return nil, err
+	}
+
+	var fwrules models.V1FirewallRules
+	err = yaml.Unmarshal([]byte(firewallRules), &fwrules)
```
Comment (mwennrich) [inline cmd/firewall.go:263]:
```suggestion
	err = yaml.Unmarshal(firewallRules, &fwrules)
```
firewallRules is already of type []byte

Comment (majst01) [reply cmd/firewall.go]: Thanks, done.

**Review by mwennrich — APPROVED** [review]

**Pull request merged**